service/src/main/kotlin/io/provenance/digitalcurrency/consortium/domain/CoinMovement.kt

```diff
@@ -11,7 +11,8 @@ import java.util.UUID
 
 const val MINT = "MINT"
 const val TRANSFER = "TRANSFER"
-const val BURN = "BURN"
+const val REDEEM_BURN = "REDEEM_BURN"
+const val DEPOSIT = "DEPOSIT"
```

Comment:
this is not backwards compatible - i'm not sure how i feel about the name "DEPOSIT" here

i just know that "BURN" no longer makes sense since transfers to bank no longer trigger a burn process automatically

anyone have any suggestions?

Reply:
credit?

Reply:
maybe this is really a REDEEM 🤷